index.html

```diff
@@ -24,65 +24,276 @@
+          <div class="canopy">
+            <p class="canopy-p">CANOPY</p>
+            <div class="round"></div>
+            <p>Back End Dev</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+          <div class="card-note">
+            <p class="daily">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="card-tags">
+            <ul class="lang">
+              <li class="lang-item">html</li>
+              <li class="lang-item">css</li>
+              <li class="lang-item1">javascript</li>
+            </ul>
+          </div>
+
+          <button type="button">See Project</button>
+        </div>
+      </div>
+      <!-- Third Card-->
+
+      <div class="work-card flex column">
+        <img src="img/IMG 2.png" alt="Tonic" />
+        <div class="work-card1 flex column">
+          <h2 class="tonic-class">Tonic</h2>
+          <div class="canopy">
+            <p class="canopy-p">CANOPY</p>
+            <div class="round"></div>
+            <p>Back End Dev</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+          <div class="card-note">
+            <p class="daily">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="card-tags">
+            <ul class="lang">
+              <li class="lang-item">html</li>
+              <li class="lang-item">css</li>
+              <li class="lang-item1">javascript</li>
+            </ul>
+          </div>
+          <div>
+            <button type="button">See Project</button>
+          </div>
+        </div>
+
+        <!-- fourth card-->
+
+        <div class="work-card flex-column">
+          <img src="img/IMG 4.png" alt="Tonic" />
+          <div class="work-card1">
+            <h2 class="tonic-class">Tonic</h2>
+            <div class="canopy">
+              <p class="canopy-p">CANOPY</p>
+              <div class="round"></div>
+              <p>Back End Dev</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+          </div>
+          <div class="card-note">
+            <p class="daily">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="card-tags">
+            <ul class="lang">
+              <li class="lang-item">html</li>
+              <li class="lang-item">css</li>
+              <li class="lang-item1">javascript</li>
+            </ul>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!--ABOUT-->
+  <section class="about">
+      <div class="about-me flex column">
+        <div class="is-me flex column">
```

Comment:
- [ ] Please as the last reviewer said Kindly ensure there is a spacing between the work section and the about me section like in the Figma design
![image](https://user-images.githubusercontent.com/74744787/219805163-6a61628f-6ea6-48f0-969f-a75b74138f2f.png)